hivemind/p2p/p2p_daemon.py

```diff
@@ -0,0 +1,290 @@
+import asyncio
+import copy
+from pathlib import Path
+import pickle
+import subprocess
+import typing as tp
+
+import google.protobuf
+from multiaddr import Multiaddr
+import hivemind.p2p.p2p_daemon_bindings.p2pclient as p2pclient
+from hivemind.p2p.p2p_daemon_bindings.datastructures import ID, StreamInfo
+
+from hivemind.utils.networking import find_open_port
+from hivemind.utils.logging import get_logger
```

Comment:
```suggestion
from multiaddr import Multiaddr

from hivemind.p2p.p2p_daemon_bindings import p2pclient
from hivemind.p2p.p2p_daemon_bindings.datastructures import ID, StreamInfo
from hivemind.utils.networking import find_open_port
from hivemind.utils.logging import get_logger
```

nit: I suggest to simplify imports and divide it to stdlib/third-party/project-wide groups.